src/utils/PaintUtils.ts

```diff
@@ -207,12 +208,16 @@ export function undo(context: CanvasRenderingContext2D,
 export function drawFromBuffer(context: CanvasRenderingContext2D,
```

Comment:
This is the function that'd need to be updated with some kind of coordinate normalization